facia-tool/public/js/utils/drag-dispatcher.js

```diff
@@ -265,7 +265,8 @@ function persist (sourceItem, newItems, sourceContext, sourceGroup, targetContex
             remove = remove ? remover(sourceContext, sourceGroup, id) : undefined;
         }
 
-        if (sourceContext !== targetContext) {
+        if (sourceContext !== targetContext ||
+            (sourceGroup.parentType === 'Collection' && sourceGroup.parent.state.fromBackfill())) {
```

Comment:
this should only be true if backfill is of type `collection`, nothing changes if type `apiquery`